pyuvdata/uvdata/tests/test_ms.py

```diff
@@ -703,7 +703,7 @@ def test_ms_extra_data_descrip(mir_uv, tmp_path):
         tb_dd.putcell(col, tb_dd.nrows() - 1, tb_dd.getcell(col, 0))
     tb_dd.close()
 
-    ms_uv.read(testfile, ignore_single_chan=False, read_weights=False)
+    ms_uv.read_ms(testfile, ignore_single_chan=False)
```

Comment:
I think this might be another rebase mistake.

Reply:
Fixed in 2d0e86b6.